### DISCUSSION TITLE
问下这是bug吗

### DISCUSSION THREAD

**Discussion by FangFuJi1991** [Meloong-Git/PCL#1520] [提问] [answered]
**精确** 地在 音乐按钮 同时按下 鼠标左右键 会导致：
![image](https://user-images.githubusercontent.com/103358064/228197838-b81c150c-0df6-4ca4-af96-6bb6664ff70d.png)
所以，这是 Bug 吗

**Comment by MoYuan-CN** [comment]
@LTCatt 是bug还是特性?

**Comment by LTCatt** [comment, marked as answer]
应该是 bug，但一般不会触发也不太好修，还是算了吧…………
